Source/Core/Core/IOS/Network/Socket.h

```diff
@@ -250,13 +262,27 @@ class WiiSockMan
   void UpdateWantDeterminism(bool want);
 
 private:
+  struct poll_command
```

Comment:
It seems like this should be part of the public interface, particularly given `AddPollCommand()` is a public function that makes use of this.

This should also be cased like `PollCommand` to be consistent with the naming convention.